bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/util/ArgumentParser.java

```diff
@@ -114,6 +114,9 @@
             if (isNameArg) {
                 isNamedArgFound = true;
                 ParamInfo info = namedArgs.get(getParamName(arg));
+                if (info == null) {
+                    throw BallerinaErrors.createError("incorrect arguments to call the 'main' function");
```

Comment:
Shall we include which arg cause the issue. @MaryamZi @praneesha Any thoughts on the error message. BTW fix looks fine for me.

Reply:
Yeah, we usually try to keep these errors similar to the compilation errors on function calls, right? Shall we say something like

`undefined parameter 'param_name'`

Reply:
+1 Better if we can make it "undefined parameter: 'param_name'" only if it doesn't create any inconsistencies with other similar occurrences.